pages/partials/questionFooter.ejs

```diff
@@ -16,18 +16,27 @@
             </button>
           <% } %>
           <% if (showGradeButton) { %>
-              <% if (question_context == 'student_homework') { %>
-                  <button class="btn btn-primary question-grade disable-on-submit order-1 mr-1" name="__action" value="grade">Save & Grade
-                      <% if ((assessment_question.tries_per_variant - variant.num_tries) > 1) { %>
-                          <span class="badge">
-                              <%= String(assessment_question.tries_per_variant - variant.num_tries) %>
-                              tries left
-                          </span>
-                      <% } %>
-                  </button>
-              <% } else { %>
-                  <button class="btn btn-primary question-grade disable-on-submit order-1 mr-1" name="__action" value="grade">Save & Grade</button>
-              <% } %>
+            <button class="btn btn-primary question-grade disable-on-submit order-1 mr-1" name="__action" value="grade">Save & Grade
+            <%  var attemptsLeft = 0, attemptsTotal = 0;
```

Comment:
Server-side code should use `let` (this was my error in the code I posted to slack code, sorry!)

```suggestion
            <%  let attemptsLeft = 0, attemptsTotal = 0;
```